src/lib/dom.js

```diff
@@ -39,6 +39,13 @@ export function getElement(id) {
 
 export function popup(url, options) {
 
+    // Condition for when dimensions (optional) for pop-up not specified
```

Comment:
since all the type checks are in the validate function it will be checked everytime. Probably I could take those two conditions out of validate and check on if options.dimensions is true. Else doing this:

if (options.dimensions.width && typeof options.dimensions.width !== 'number') {
        throw new Error(`[${options.tag}] Expected options.dimensions.width to be a number`);
    }

will trow errors